src/pages/linux/some-usages-for-linux/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Some Usages for Linux
+How Linux Is Used
```

Comment:
Change this line to:
```
title: How Linux Is Used
```

Reply:
Thank you! I think I fixed it. If not please let me know (trying to learn Github :D )

Reply:
No problem :)
Yeah, you fixed it, Travis CI ate this code without any errors, ahahah! I'm also trying to learn GitHub :blush: 

Greetings, Daria